src/frontend/js/utils/ContractHelper/index.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { createIntl } from 'react-intl';
+import { ContractFactory } from 'utils/test/factories/joanie';
+import { ContractState } from 'types/Joanie';
+import { ContractHelper, ContractStatePoV } from '.';
+
+const unsignedContractFactory = ContractFactory({
```

Comment:
When i first read i would have feel more confortable wiith `UnsignedContractFactory`, but it's a instanciated factories so may be not.